bibliopixel/VERSION

```diff
@@ -1 +0,0 @@
-3.4.18
```

Comment:
Urg, I'm not looking forward to seeing how we edit this value automatically if it moves to a Python file... :-/

Reply:
Thinking more about it, it does seem logical to have `VERSION` and `LICENSE` as their own files in the package, and do what you do already by loading them into `__init__.py`.